examples/LoggingExtensions/Program.cs

```diff
@@ -22,7 +22,7 @@
 
 var resourceBuilder = ResourceBuilder.CreateDefault().AddService("Examples.LoggingExtensions");
 
-var openTelemetryLoggerProvider = new OpenTelemetryLoggerProvider(options =>
+var openTelemetryLoggerProvider = OpenTelemetryLoggerProvider.Create(options =>
```

Comment:
Would a static method on `Sdk` make sense? Like `Sdk.CreateMeterProviderBuilder` and `Sdk.CreateTracerProviderBuilder`

Reply:
Good idea I like it! At the moment the mechanics are slightly different in that there is no `Build` method for `OpenTelemetryLoggerProvider`. I could introduce a small class `LoggerProviderBuilder` so it matches more closely what we have to metrics + traces. Thoughts on that?

Reply:
If folks aren't thrilled by that idea, then I think `Sdk.CreateLoggerProvider` (i.e., w/o `Builder`) wouldn't be terrible. 

But, consistency would have a nice feel to it, so my vote is a simple class enabling `Sdk.CreateLoggerProviderBuilder(...).Build()`

Reply:
@alanwest OK this now has more of a builder pattern. Updated the description. LMK what you think!